src/commcare_cloud/commcare_cloud.py

```diff
@@ -150,9 +150,70 @@ def make_parser(available_envs, formatter_class=RawTextHelpFormatter,
     return parser, subparsers, commands
 
 
+def compile_changelog():
```

Comment:
I would move all of this code plus the `MakeChangelog` to a new file `manage_commcare_cloud/make_changelog.py`. This file, which is the entrypoint for `commcare-cloud` tool, doesn't feel like an appropriate place to put the logic behind autogenerating a piece of documentation.

As for moving `MakeChangelog` (and this logic) to its own file rather than sharing it with `make_docs.py`, that's just to emphasize that while these two commands are _analogous_ (they both autogenerate a piece of documentation), they don't actually share much if any of their implementation logic and are actually quite separate from one another.

Reply:
https://github.com/dimagi/commcare-cloud/pull/1979/commits/14ec7abd7521f11bb0e6c6fbb6bd15ca9eb25794